api/UserController.js

```diff
@@ -129,6 +130,60 @@ exports.updateUser = function(req, res) {
         })
 };
 
+exports.forgotPassword = function(req, res) {
+    console.log(req.body);
+    return db.User.findOne({
+        where: {email: req.body.email}
+    }).then(user => {
+        if (!user) {
+            return util.detailErrorResponse(res, 404, 'User not found.');
+        } else {
+            emailutil.sendResetPasswordEmail(
+                            user.dataValues.email, 
+                            'info@boardgameweekend.party',
+                            'BoardGameCompanion', 
+                            user.dataValues.name, 
+                            'http://localhost:8080/#/auth/reset_password?token=' + 
```

Comment:
Do I change to staging address now? How can we abstract that?

Reply:
Where can I put the env variable? :) UserController or user.js? Or none of them? :p